src/app/file/jpeg_format.cpp

```diff
@@ -177,12 +178,38 @@ bool JpegFormat::onLoad(FileOp* fop)
   // Start decompressor.
   jpeg_start_decompress(&dinfo);
 
+  // Get orientation:
+  TinyEXIF::EXIFInfo info;
+  int orientation = 0;
+#if LAF_WINDOWS
+  const int tempBufSize = (int)std::ftell(file);
+  rewind(file);
+  char *tempBuf = (char *)malloc(sizeof(char) * tempBufSize);
+  if (!tempBuf) {
+    std::free(tempBuf);
```

Comment:
If tempBuf is null there is no point in trying to deallocate it. Anyway, see my other comment.